### PR TITLE
Normie Headings closes #3000

### LINKED ISSUE #3000: BIIIIIIIG HEADINGS
### issue posted by @garthkoyle 

The Headings are Really BIG. Do we plan to use these styles or make something a little smaller so they don't take up so much screen real estate? https://www.screencast.com/t/E5rmNSYqW

### PR DESCRIPTION
**Pull request by tn3rb**
see #3000

- reduces heading sizes and increases font color contrast

- tweaks some margin sizes on various screen sizes

### CODE REVIEW

File: assets/src/application/ui/layout/entityList/style.scss [hunk 1]
```diff
@@ -78,6 +81,8 @@
 	}
 
 	+ .ee-entity-list {
-		margin-top: calc(var(--ee-margin-big) * -1);
+		@include min1600px {
+			margin-top: calc(var(--ee-margin-big) * -1);
+		}
 	}
```
Comment (tn3rb) [inline assets/src/application/ui/layout/entityList/style.scss:87]: only reduces space between entity lists on really wide screens

File: assets/src/application/ui/input/Button/ButtonRow/style.scss [hunk 2]
```diff
@@ -29,20 +28,27 @@
 		}
 	}
 
-	@include max782px {
-		margin: var(--ee-margin-defult) 0;
-	}
+	ee-form + & {
+		margin: var(--ee-margin-big) 0;
```
Comment (tn3rb) [inline assets/src/application/ui/input/Button/ButtonRow/style.scss]: only adds space around button rows on forms

File: assets/src/application/ui/layout/entityList/style.scss (hunk 1 above)
Comment (nicolad) [reply assets/src/application/ui/layout/entityList/style.scss:87]: smart

**Comment by manzoorwanijk** [comment]
They look great

demo

File: assets/src/application/ui/styles/mixins/breakpoints.scss [hunk 3]
```diff
@@ -108,6 +108,18 @@ $breakpointsMapping: (
...
+	@include breakpoint(1600px, max) {
+		@content;
+	}
+}
+
+@mixin min1600px {
+	@include breakpoint(1600px, min) {
+		@content;
+	}
+}
```
Comment (nicolad) [inline assets/src/application/ui/styles/mixins/breakpoints.scss:121]: I am glad that you are using and extending these mixins :+1:

**Comment by nicolad** [comment]
Thanks @manzoorwanijk for those screenies, looks amazing :+1:

**Pull request merged**